Dockerfile

```diff
@@ -0,0 +1,30 @@
+# UBI with Python 3.8 version, see base Dockerfile here:
+# https://catalog.redhat.com/software/containers/ubi8/python-38/5dde9cacbed8bd164a0af24a?gti-tabs=unauthenticated&container-tabs=dockerfile
+FROM registry.access.redhat.com/ubi8/python-38:latest
+
+LABEL MAINTAINER "Avishkar Gupta <avgupta@redhat.com>"
+
+EXPOSE 6006
+
+# Upgrade dependency management itself.
+RUN pip install --upgrade pip pipenv
+
+# Copy the sources to the app root, see base image Dockerfile for more details.
+WORKDIR /opt/app-root/src/
+
+COPY ./src/ .
+
+# Copy the locked env since this is a production build.
+COPY ./Pipfile Pipfile
```

Comment:
Is this required as only the lockfile will be used.

Reply:
Yes it otherwise raises a "Pipefile not found" error, it looks for both even if it uses the one.